src/DataGrid.tsx

```diff
@@ -1182,6 +1183,7 @@ function DataGrid<R, SR, K extends Key>(
               );
             })}
             <RowSelectionChangeProvider value={selectRowLatest}>
+              {/* eslint-disable-next-line react-compiler/react-compiler */}
```

Comment:
Don't know which line it is complaining about

Reply:
Probably the `setDraggedOverRowIdx` call?

Reply:
I tried removing it and it was still showing the same warning